src/Proto.Cluster/Partition/PartitionIdentityRebalanceWorker.cs

```diff
@@ -94,7 +94,7 @@ private Task OnPartitionCompleted(PartitionCompleted response, IContext context)
         private Task OnPartitionFailed(PartitionFailed response, IContext context)
         {
             Logger.LogWarning("[PartitionIdentity] Retrying member {Member}, failed with {Reason}", response.MemberAddress, response.Reason);
-            StartRebalanceFromMember(_request!, context, response.MemberAddress);
+            context.ReenterAfter(Task.Delay(200, _cancellationToken), () => StartRebalanceFromMember(_request!, context, response.MemberAddress));
```

Comment:
Can we have some form of RequestAsync initialization here?
e.g. we send a request and if we get a proper answer back, we continue with the rebalance.
if we get timeout. do exponential backoff?

Reply:
Alternative suggestion. We already have the reason for the failure (deadletter or timeout). If it's a deadletter it makes sense to wait before a retry, but if it is a timeout it would make sense to retry instantly..